tests/test_forward_ref.py

```diff
@@ -1365,3 +1365,78 @@ class Model(BaseModel, A):
         ReturnedModel = inner()  # noqa: F841
 
     assert module_1.ReturnedModel.__pydantic_complete__ is False
+
+
+# Tests related to forward annotations evaluation coupled with PEP 695 generic syntax:
```

Comment:
Is this needed, given the skipifs below?

Reply:
Fine if you want to keep it, just wanted to check :)

Reply:
The comment acts as some kind of "grouping" of tests (which in reality is a sign that we should split our test files :sweat_smile:), and at some point (in a really long time) the skipifs will be removed but the comment should remains